extensions/azcli/src/typings/az-ext.d.ts

```diff
@@ -258,104 +257,85 @@ declare module 'azdata-ext' {
 		}
 	}
 
-	export interface AzdataOutput<R> {
-		logs: string[],
-		result: R,
-		stderr: string[],
-		stdout: string[],
+	export interface AzOutput<R> {
```

Comment:
I expect az outputs will be harder to parse than azdata.  azdata --output json used to return stdout, stderr, and the result separately.  That allowed printing human readable messages while also returning a computer parsable result.  But I noticed az is just combining everything.

Reply:
So far, assigning stdout to the output of az commands has worked for data controller connection. Perhaps for this PR we can just leave it like this and the AzOutput will be subject to change later on if this does not work for other commands?

Reply:
Yeah I think that's a good idea. Doing fixes like that in separate PRs will make it easier to validate the changes being made.